contracts/token/psp721/tests/tests.rs

```diff
@@ -82,9 +84,11 @@ mod tests {
         }
     }
 
-    impl IPSP721Mint for PSP721Struct {}
+    impl PSP721Mintable for PSP721Struct {}
+
+    impl PSP721Burnable for PSP721Struct {}
 
-    impl IPSP721Metadata for PSP721Struct {}
+    impl PSP721Metadata for PSP721Struct {}
 
```

Comment:
should any tests for `mint` and `burn` be added here?

Reply:
there was a trait PSP721Mint which implemetned burn and mint and it was implemented in the tests, so you can mint and burn in the test. So the tests for mint and burn already existed, but not for the extension trait